polkadot/runtime/src/lib.rs

```diff
@@ -92,6 +92,8 @@ pub type Address = staking::Address<Concrete>;
 pub type BlockId = generic::BlockId<Block>;
 /// Unchecked extrinsic type as expected by this runtime.
 pub type UncheckedExtrinsic = generic::UncheckedExtrinsic<Address, Index, Call, Signature>;
+/// Checked extrinsic type as expected by this runtime.
+pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, Index, Call>;
```

Comment:
Prefer to derive as `<UncheckedExtrinsic as Checkable>::Checked`, otherwise we might end up with crazy error messages when the types diverge.

Reply:
changed this because `<UncheckedExtrinsic as Checkable>::Checked` https://github.com/paritytech/polkadot/pull/287/files#diff-c7bae7b0ef067fe57fc5ad9bab233f29L57 no longer compiles as `Checkable` now takes the type param `Context`. could change this to `pub type CheckedExtrinsic<Context> = <UncheckedExtrinsic as Checkable<Context>>::Checked;` but then the type param would be required all throughout https://github.com/paritytech/polkadot/blob/master/polkadot/transaction-pool/src/lib.rs since https://github.com/paritytech/polkadot/blob/master/polkadot/transaction-pool/src/lib.rs#L64 and then for everything that uses `VerifiedTransaction`. how should i proceed on this?

Reply:
fixed in https://github.com/paritytech/polkadot/pull/287/commits/21676fe8a80c52fb68c8b1d0e2f5dd5811f2c529